src/net_processing.cpp

```diff
@@ -4689,7 +4689,8 @@ bool PeerManagerImpl::SendMessages(CNode* pto)
         //
         // Message: feefilter
         //
-        if (pto->m_tx_relay != nullptr && pto->GetCommonVersion() >= FEEFILTER_VERSION && gArgs.GetBoolArg("-feefilter", DEFAULT_FEEFILTER) &&
+        if (pto->m_tx_relay != nullptr && !m_ignore_incoming_txs && pto->GetCommonVersion() >= FEEFILTER_VERSION &&
```

Comment:
It might be good to split each `&&` into its own line, to clarify that each condition needs to be fulfilled independently

Reply:
Done.